OpenKh.Tools.ModsManager/Views/InstallModView.xaml.cs

```diff
@@ -13,7 +13,8 @@ public partial class InstallModView : Window
     {
         private static readonly IEnumerable<FileDialogFilter> _zipFilter = FileDialogFilterComposer
             .Compose()
-            .AddExtensions("OpenKH mod as ZIP file", "zip");
+            .AddExtensions("OpenKH ZIP Mod", "zip")
```

Comment:
Might consider something like

```cs
.AddExtensions("Mod Formats", "zip", "kh2pcpatch");
```

so users don't have to know to use the chooser to switch between them

Reply:
This is intentional to a degree. Will not be altering it as of now.

Reply:
Reasons? I thought this was broken at first until I saw the chooser, and I'm going to be more tech savvy than most players.